src/components/demo/DOITooltipDemo.tsx

```diff
@@ -88,7 +136,7 @@ const DOITooltipDemo = (props: Props) => {
             <button type="button" style={{ color: 'blue', cursor: 'pointer' }} ref={targetRef} onClick={() => TriggerTooltip()}> {children} </button>
 
             <div id="disscoTooltip" className={`tooltip ${active && 'active'}`} ref={DOITooltipRef} style={offsetStyles}>
-                {record.data ?
+                {record?.data ?
```

Comment:
?

Reply:
What does this do here

Reply:
Normally you use to say 'ok, if this property is there, use this one', it can be undefined, you are not sure.
In this case I removed it again as it already is a check if the data property is there, so it would be duplicate.